python/tvm/arith/iter_affine_map.py

```diff
@@ -117,14 +117,14 @@ def detect_iter_map(
 
     Returns
     -------
-    results : List[IterSumExpr]
+    results : IterMapResult
         The iter map matching result.
-        Empty array if no match can be found.
+        The result's .indices is empty array if no match can be found.
 
     """
     return _ffi_api.DetectIterMap(
         indices, input_iters, predicate, require_bijective, simplify_trivial_iterators
-    )
+    ).indices
```

Comment:
I think this line disagrees with the return type in the docstring.  We should either remove the `.indices` to return a `IterMapResult`, or revert the docstring changes to describe returning a `IterMapResult`.

Reply:
I agree, I am trying to rewrite cases in test_arith_iter_affine_map.py